mpi.rst

```diff
@@ -257,6 +259,24 @@ If the host MPI is Open MPI, the definition file looks like:
 
       mpirun -np 2 -mca pml ucx -x UCX_TLS=sysv,ib apptainer exec $MY_CONTAINER ./a.out
 
+Using --sharens mode
+====================
+
+Default in unprivileged user namespace mode. Each process spawned by MPI runs in its own user namespace. This will 
+bring problems as observed :ref:`in this issue <mpi-issue>`. The new ``--sharens`` flag will 
+move all spawned processes into the same user namespace, similar to what is done with {Project} SUID mode. 
+
+.. code::
+
+   mpirun -np 2 apptainer exec --sharens $MY_CONTAINER ./a.out
```

Comment:
For consistency please change `apptainer` to `{command}`.

Reply:
thanks! reflected in the latest commit.